blocks/library/image/index.js

```diff
@@ -177,8 +177,6 @@ export const settings = {
 
 		if ( width ) {
 			figureStyle = { width };
-		} else if ( align === 'left' || align === 'right' ) {
-			figureStyle = { maxWidth: '50%' };
```

Comment:
I know there have been discussions about whether to add this or not, not sure about the conclusion here though @jasmussen

Reply:
Yep — mostly I'm in the camp of _the fewer inline styles, the better_, because themes can't override them without using `!important`. So we should be very careful before we add inline styles like this.